app/Nova/Filters/IsOld.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Nova\Filters;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Filters\Filter;
+
+class IsOld extends Filter
+{
+    /**
+     * The filter's component.
+     *
+     * @var string
+     */
+    public $component = 'select-filter';
+
+    /**
+     * Apply the filter to the given query.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  mixed  $value
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function apply(Request $request, $query, $value)
+    {
+        return $query->where([['isOld',$value['isOld']],['user_id',$value['id']]]);
```

Comment:
filter should do one thing if you need multible where (unrelated to the current filter) add new one